src/content/en/updates/2019/07/web-components-time-to-upgrade.md

```diff
@@ -0,0 +1,219 @@
+       larger than the other polyfills, and has greater performance impact. Only use this bundle if 
+       you need shadow DOM support.
+   </td>
+  </tr>
+  <tr>
+   <td><code>webcomponents-lite-min.js</code>
+   </td>
+   <td>Includes all polyfills except for shadow DOM.
+
+Note: Polymer 1.x users should choose this bundle, since Shadow DOM emulation was included directly 
+in the Polymer library. Polymer 2.x users need a different version of the polyfills. See 
+<a href="https://www.polymer-project.org/blog/2018-10-02-webcomponents-v0-deprecations">the Polymer 
+Project blog post</a> for details.
+   </td>
+  </tr>
+</table>
+
+
+There are also individual polyfills available as part of the Web Components polyfill package. Using 
+individual polyfills separately is an advanced topic, not covered here.
+
+To load the polyfills unconditionally:
+
+```html
+<script src="/bower_components/webcomponents/webcomponentsjs/webcomponents-lite-min.js">
+```
+
+Or:
+
+```html
+<script src="/bower_components/webcomponents/webcomponentsjs/webcomponents-min.js">
+```
+
+If you installed the polyfills directly from GitHub, you'll need to supply the path where you 
+installed the polyfills.
+
+If you conditionally load the polyfills based on feature detection, your site should continue to 
+work when v0 support is removed.
+
+If you conditionally load the polyfills using browser sniffing (that is, loading the polyfills on 
+non-Chrome browsers), your site will fail when v0 support is removed from Chrome. 
+
+## Help! I don't know what APIs I'm using! {#help-i-don't-know-what-apis-i'm-using}
+
+If you don't know whether you're using any of the v0 APIs--or _which_ APIs you're using, you can 
+check the console output in Chrome.
+
+1.  If you previously started Chrome with the flags to disable the v0 APIs, close Chrome and restart
+    it normally.
+1.  Open a new Chrome tab and load your site.
+1.  Press Control+Shift+J (Windows, Linux, Chrome OS) or Command+Option+J (Mac) to open the DevTools
+    **Console**. 
+1.  Check the console output for deprecation messages. If there's a lot of console output, enter 
+    "Deprecation" in the **Filter** box.
+
+<img src="/web/updates/images/2019/07/web-components-console-deprecations.png" 
+    alt="Console window showing deprecation warnings">
+
+You should see deprecation messages for each v0 API you're using. The output above shows messages 
+for HTML Imports, custom elements v0, and shadow DOM v0.
+
+If you're using one or more of these APIs, see [Use the v0 polyfills](#use-the-v0-polyfills).
+
+
+## Next steps: moving off of v0 {#next-steps-moving-off-of-v0}
+
+Making sure the v0 polyfills are getting loaded should ensure your site keeps working when the  v0 
+APIs are removed. Moving forward, we recommend moving off of the v0 APIs entirely so you can take 
+advantage of cross-platform, native support for the Web Components v1 APIs.
```

Comment:
Given its importance, this last sentence needs to be punchier. Maybe:

We recommend dropping v0 for the cross-platform, native support of Web Components v1.

Reply:
I want to avoid saying "dropping" lest someone read it as "dropping web components." Agree on punchier, maybe "Moving forward, we recommend moving to Web Components v1, which is broadly supported."

Reply:
Sound better but drop 'moving forward'. It doesn't add anything.

Reply:
👍